pkgs/applications/virtualization/runc/default.nix

```diff
@@ -0,0 +1,64 @@
+{ stdenv, fetchFromGitHub, go-md2man
+, go, pkgconfig, libapparmor, apparmor-parser, libseccomp }:
+
+with stdenv.lib;
+
+stdenv.mkDerivation rec {
+  name = "runc-${rev}";
+  rev = "cc29e3dd"; # this version is used by docker
```

Comment:
Well apparently stable releases of runc are older than commit, i will replace with commit date.